internal/cli/option/flag.go

```diff
@@ -4,6 +4,12 @@ import "github.com/spf13/pflag"
 
 type FlagFn func(*pflag.FlagSet)
 
+func Filenames(v *[]string) FlagFn {
+	return func(fs *pflag.FlagSet) {
+		fs.StringSliceVarP(v, "filename", "f", nil, "Filename to use to create the resource")
```

Comment:
Since, I believe, directories are supported now, and _I think_ URLs are also since you're re-using parts of `kubectl` for this, this might be more complete wording:

> Filename, directory, or URL to files to use to create the resource

☝️ Cribbed from `kubectl` as well

Reply:
Also, I  notice this flag is used by all three of create, update, and delete. "Filename to use to create the resource" seems like the wrong message for update and delete sub-commands.